src/app/screens/ProfileScreen/tabs/PersonalDetails.tsx

```diff
@@ -65,20 +64,27 @@ const PersonalDetails = (props: IPersonalDetailsData) => {
       </Card>
 
       <Card title="Emergency Contact Details">
-        <DetailRow label="Name" value={emergencyContactDetails[0]?.name} />
-        <DetailRow
-          label="Relation"
-          value={emergencyContactDetails[0]?.relation}
-        />
-        <DetailRow
-          label="Phone No"
-          value={emergencyContactDetails[0]?.phoneNumber}
-        />
+        {emergencyContactDetails?.map(
```

Comment:
What is our expected behaviour when `emergencyContactDetails` is an empty list ?
- Card is hidden
- Show empty card with title 
- Show Card with title and a message